packages/suite/src/views/settings/SettingsDevice/WipeCode.tsx

```diff
@@ -0,0 +1,53 @@
+import { analytics, EventType } from '@trezor/suite-analytics';
+
+import { changeWipeCode } from 'src/actions/settings/deviceSettingsActions';
+import {
+    ActionButton,
+    ActionColumn,
+    SectionItem,
+    TextColumn,
+    Translation,
+} from 'src/components/suite';
+import { SettingsAnchor } from 'src/constants/suite/anchors';
+import { useDispatch } from 'src/hooks/suite';
+import { useAnchor } from 'src/hooks/suite/useAnchor';
+
+interface ChangePinProps {
+    isDeviceLocked: boolean;
+}
+
+export const WipeCode = ({ isDeviceLocked }: ChangePinProps) => {
+    const dispatch = useDispatch();
+    const { anchorRef, shouldHighlight } = useAnchor(SettingsAnchor.WipeCode);
+
+    const handleClick = () => {
+        dispatch(changeWipeCode({ remove: false }));
+        analytics.report({
+            type: EventType.SettingsDeviceChangePin,
+        });
+    };
+
+    return (
+        <SectionItem
+            data-test="@settings/device/wipe-code"
+            ref={anchorRef}
+            shouldHighlight={shouldHighlight}
+        >
+            <TextColumn
+                title={<Translation id="TR_DEVICE_SETTINGS_WIPE_CODE_TITLE" />}
+                description={<Translation id="TR_DEVICE_SETTINGS_WIPE_CODE_DESC" />}
+                buttonLink="https://trezor.io/learn/a/create-wipe-code-to-erase-device"
```

Comment:
this should be listed here https://github.com/trezor/trezor-suite/blob/fix/connect-webexplorer-to-allow-more-than-one-request/packages/urls/src/urls.ts#L0-L1